src/js/lib/client/src/index.js

```diff
@@ -383,6 +382,15 @@ function urlParamsFromObject(obj) {
 function createWindow() {
     let webPreferences     = secureWebPreferences()
     webPreferences.preload = path.join(root,'preload.js')
+
+    // Note: this is a conditional default. We want borders on Win/Linux and borderless on Mac.
+    if (args.frame === undefined && process.platform === 'darwin') {
+        args.frame = false
+    }
+    if (args.frame === undefined && process.platform !== 'darwin') {
+        args.frame = true
+    }
```

Comment:
this could be refactored to something simpler:
```js
if (args.frame === undefined) {
    args.frame = proces.platform !== 'darwin'
}

```

Reply:
Btw, the placeent of this code is really bad. Why function `createWindow` is responsible for mutating command line arguments and adding some defaults there? Function `createWindow` should just create a window.